### PR TITLE
7.0 mgmtsystem nonconformity description

### PR DESCRIPTION
**Pull request by foutoucour**
Module to remove the spam description can produce in the view list and in the header of the form view

### CODE REVIEW

File: mgmtsystem_nonconformity_description/mgmtsystem_nonconformity.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+##############################################################################
+
+from openerp.tools.translate import _
+from openerp.osv import fields, orm
+
+
+class mgmtsystem_nonconformity(orm.Model):
+    """ Add a field name to the nonconformity. """
+    _inherit = "mgmtsystem.nonconformity"
+    _rec_name = "ref"
```
Comment (bwrsandman) [inline mgmtsystem_nonconformity_description/mgmtsystem_nonconformity.py]: Why not add set ref as _rec_name directly in the `mgmtsystem_nonconformity` module?

**Comment by dreispt** [comment]
Why a new module? Wouldn't it be best to modify `mgmtsystem_nonconformity` directly?

A question: would it be inappropriate to have a title and a description, as is usual in other models (Project Issues, for example)?

**Comment by bwrsandman** [comment]
Same question as @dreispt

**Comment by max3903** [comment]
:-1:

**Comment by foutoucour** [comment]
Merging the update in the mgmsystem_nonconformity as suggested.

**Comment by foutoucour** [comment]
Merging the update in the mgmsystem_nonconformity as suggested.

**Pull request closed without merging**